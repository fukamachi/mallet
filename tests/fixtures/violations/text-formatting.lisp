(defpackage #:malvolio-test-formatting
  (:use #:cl))
(in-package #:malvolio-test-formatting)

;; This file has various text-level formatting violations

;; Trailing whitespace on this line
(defun foo ()
  (+ 1 2))

;; Tab character on next line
	(defun bar ()
  (+ 3 4))

;; Too many consecutive blank lines follow




(defun baz ()
  (+ 5 6))

;; File ends without newline - this is the last line