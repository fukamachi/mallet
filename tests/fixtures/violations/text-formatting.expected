# Format: line:column rule-name severity
12:0 no-tabs warning
18:0 consecutive-blank-lines warning
22:0 final-newline warning
